src/test/java/TestChiralRead.java

```diff
@@ -1268,5 +1313,646 @@ public void testQueryStructureSimpleTetrahedralStereoDoesntError() throws Except
+				.filter(ca->ca.getChirality()!=Chirality.Non_Chiral)
+				.map(ca->ca.getChirality())
+				.collect(Collectors.toList());
+		assertTrue(listChi.size() > 0);
+		for (Chirality chirality : listChi) {
+			System.out.printf("chirality: ");
+		}*/
+	}
+
+	@Test
+	public void testSlowChiralityTrueMultiple() throws IOException {
+		List<TestMol> mols = Arrays.asList(
+				new TestMol("large_polymer1", "large_polymer1.mol", 3),
+				new TestMol("614e808b-234a-476b-ac60-98ea42d6c6c5", "614e808b-234a-476b-ac60-98ea42d6c6c5.mol", 0),
+				new TestMol("c2dd8ca2-9ff7-4144-bcc7-77684294e24b", "c2dd8ca2-9ff7-4144-bcc7-77684294e24b.mol", 0),
+				new TestMol("c09aefcd-e985-4b8e-aa94-23a8f616228a", "c09aefcd-e985-4b8e-aa94-23a8f616228a.mol", 0),
+				new TestMol("921c6da6-152f-4da9-b684-579442ff6ad5", "921c6da6-152f-4da9-b684-579442ff6ad5.mol", 0),
+				new TestMol("452b31d5-7993-4cdf-b11c-243e433d4e34", "452b31d5-7993-4cdf-b11c-243e433d4e34.mol", 0),
+				new TestMol("9V35WW05U3", "9V35WW05U3.mol", 0),
+				new TestMol("79d19ec1-d053-44c3-a6c3-a6f0d0cdeea6", "79d19ec1-d053-44c3-a6c3-a6f0d0cdeea6.mol", 0),
+				new TestMol("46091f88-fd25-4ca9-8e37-fced5329be5a", "46091f88-fd25-4ca9-8e37-fced5329be5a.mol", 0),
+				new TestMol("benzoic acid", "benzoic.acid.mol", 0),
+				new TestMol("R 1-Phenylethanol", "1-Phenylethanol-R.mol", 1),
+				new TestMol("S 1-Phenylethanol", "1-Phenylethanol-S.mol", 1),
+				new TestMol("racemic 1-Phenylethanol", "1-Phenylethanol-racemic.mol", 1),
+				new TestMol("anthracene", "anthracene.mol", 0),
+				new TestMol("perylene", "perylene.mol", 0),
+				new TestMol("coronene", "coronene.mol", 0),
+				new TestMol("dioxocoronene", "dioxocoronene.mol", 0),
+				new TestMol("tetraoxocoronene", "tetraoxocoronene.mol", 0),
+				new TestMol("octaoxocoronene", "octaoxocoronene.mol", 0),
+				new TestMol("ovalene", "ovalene.mol", 0),
+				new TestMol("double-ovalene", "double-ovalene.mol", 0),
+				new TestMol("biggish1", "biggish1.mol", 0),
+				new TestMol("biggish2", "biggish2.mol", 0),
+				new TestMol("biggish3", "biggish3.mol", 4),
+				new TestMol("biggish4", "biggish4.mol", 12),
+				new TestMol("biggish5", "biggish5.mol", 12),
+				new TestMol("biggish6", "biggish6.mol", 12),
+				new TestMol("biggish7", "biggish7.mol", 0),
+				new TestMol("biggish8", "biggish8.mol", 0),
+				new TestMol("biggish9", "biggish9.mol", 4),
+				new TestMol("biggish10", "biggish10.mol", 4),
+				new TestMol("small_symmetric","small_symmetric.mol", 4)
+		);
+		mols.forEach(m->{
+			try{
+				System.out.printf("about to test %s\n", m.molName);
+				assertTrue(testOneMol(m));
+			}catch (IOException ex){
+				System.err.printf("Error processing mol %s - %s\n",m.molName, ex.getMessage());
+				fail("test fails!");
+			}});
+	}
+
+	private boolean testOneMol(TestMol testMol) throws IOException {
+		String userDirectory = new File("").getAbsolutePath();
+		String fileName =userDirectory + "/src/test/resources/mols/" + testMol.molfileName;
```

Comment:
In general I think this can get a bit dodgy based on what OS/classloaders/etc are being used. I think the more kosher way to do it is like this:

```
 String molfileText = IOUtils.toString(
      this.getClass().getResourceAsStream("mols/" + testMol.molfileName),
      "UTF-8"
    );
    Chemical c1=Chemical.parse(molfileText );
    
```